### ISSUE TITLE
Jupyter Notebook python cell format issue

### ISSUE THREAD

**Issue by Theakayuki** [dotenv-org/dotenv-vscode#63]
![image](https://user-images.githubusercontent.com/58452606/196066906-379467ac-1169-4887-aa6e-4c0ad8a4cde9.png)

When installed and edited a jupyter notebook will change the python format to .env its only the formating as the language and the cell still run fine. but all the python auto completion and formating gets striped.

Expected to not interfere with formatting or auto complete. like how it is not installed:

![image](https://user-images.githubusercontent.com/58452606/196067319-5a347598-681e-42d1-823d-756468df1b95.png)

**Comment by motdotla** [comment]
Another individual reported this to support@dotenv.org. Additional information:

```
I recently installed your extension for Visual Studio Code and everything was working fine until I had to open a Jupyter notebook.
It seems that there is some kind of incompatibility because the VS Code started to associate the code cells with ".env" language.
Worst of all is when you try to change it manually, the menu tells you that Python is the current language and if you select it nothing happens. it is important to clarify that there is no problem for the rest of the languages. I attached two screenshots as an example.

Is there a way to configure the extension to ignore ".ipynb" files or something similar?
```

**Comment by leranjun** [comment]
Can reproduce. Using extension version 0.21.0.

**Comment by ConnorJennison** [comment]
Thank you for posting this and sorry it's taken so long to address. I pushed a fix for #66 and from my testing it should also address this issue. I will follow up when it is published and let you know

**Comment by ConnorJennison** [comment]
v0.22.0 has been released and this should be solved now, please let me know if you run into issues with it but will close for now!
